src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,40 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final int OPERATION = 0;
+    private static final int AMOUNT = 1;
+
     public void getStatistic(String fromFileName, String toFileName) {
 
+        try (BufferedReader bufferedReader = new BufferedReader(new FileReader(fromFileName));
+                 BufferedWriter bufferedWriter = new BufferedWriter(new FileWriter(toFileName))) {
+            int supply = 0;
+            int buy = 0;
+            String line;
+            while ((line = bufferedReader.readLine()) != null) {
+                String[] files = line.split(",");
+                if (files[OPERATION].equals("supply")) {
```

Comment:
This strings you can add to constant 
![image](https://github.com/mate-academy/jv-work-with-file/assets/63095184/59431f54-c2c2-4216-8daa-5122af3781cd)